ui-components/src/page_components/listing/ListingCard.stories.tsx

```diff
@@ -77,3 +79,76 @@ export const withStackedTable = () => {
     />
   )
 }
+
+export const NoContent = () => {
+  return (
+    <ListingCard
+      imageCardProps={{
+        imageUrl:
+          "https://res.cloudinary.com/exygy/image/upload/w_1302,c_limit,q_65/dev/old-oakland-kim-cole-3-1024x768_gdwmzt.jpg",
+        subtitle: "subtitle",
+        title: "title",
+        href: "listing-link",
+      }}
+    />
+  )
+}
+
+const exampleCustomContent = () => {
+  const getHeader = (header: string) => {
+    return (
+      <div
+        className={
```

Comment:
Would love to pull our combinations of text classes like these more parent classes so we can standardize more variations of "text-header-1" etc